Formula/mono-libgdiplus.rb

```diff
@@ -1,10 +1,13 @@
 class MonoLibgdiplus < Formula
   desc "GDI+-compatible API on non-Windows operating systems"
   homepage "https://www.mono-project.com/docs/gui/libgdiplus/"
-  url "https://github.com/mono/libgdiplus/archive/6.0.5.tar.gz"
-  sha256 "1fd034f4b636214cc24e94c563cd10b3f3444d9f0660927b60e63fd4131d97fa"
+  url "https://download.mono-project.com/sources/libgdiplus/libgdiplus-6.1.tar.gz"
+  sha256 "97d5a83d6d6d8f96c27fb7626f4ae11d3b38bc88a1726b4466aeb91451f3255b"
   license "MIT"
-  revision 1
+
+  livecheck do
+    url "https://github.com/mono/libgdiplus.git"
+  end
```

Comment:
cc @samford

Reply:
```suggestion
  livecheck do
    url "https://download.mono-project.com/sources/libgdiplus/"
    regex(/href=.*?libgdiplus[._-]v?(\d+(?:\.\d+)+)\.t/i)
  end
```

There's something to be said for checking the directory listing page where the `stable` archive is found, as it serves as the download page for the related source files and aligns with the `stable` source. With checking the Git repository, there's a possibility that upstream may tag a new version before the related source file is published on the first-party site and that could cause problems.